### PR TITLE
Saving progess using local storage

### PR DESCRIPTION
**Pull request by sujith-rek**
## Summary

active code changes that facilitate progress saving
Solves the issue #251 

## Checklist

- [X] Variables, functions, and comments are translated into Spanish
- [X] Functions follow underscore notation
- [X] Spell check done & typos fixed
- [X] All Python code is PEP8 compliant
- [X] Test coverage with Playwright implemented; locators are Python code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots
- Before clicking Save & Run
image

- After Save & Run
image

- Video of the process: 
ls.webm

### CODE REVIEW

**Review by NicolasSandoval — APPROVED** [review]
LGTM!

File: tests/test_TWP42.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def test_l42_2(page):
     page.goto("lectures/TWP42/TWP42_2.html")
 
     page.click("text=4​ >> pre[role=\"presentation\"]")
-    page.fill("text=RunLoad History21 1from sqlite3 import connect2con = connect('alumnos.bd')3cur = >> textarea", "cur.execute('''create table alumnos(login_id varchar(8),pass integer)''')")
+    page.fill("text=RunResetLoad History21 1from sqlite3 import connect2con = connect('alumnos.bd')3cur = >> textarea", "cur.execute('''create table alumnos(login_id varchar(8),pass integer)''')")
```
Comment (reingart) [inline tests/test_TWP42.py:67]: Does this change in the locator break any other test?

**Review by reingart — CHANGES_REQUESTED** [review]
Great work!

Please see my comments so we can merge this PR.

File: tests/test_TWP42.py (hunk 1 above)
Comment (sujith-rek) [reply tests/test_TWP42.py:67]: no, all the other tests working fine

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/angelasofiaremolinagutierrez/RunestoneComponents.git@master#runestone
+git+https://github.com/sujith-rek/RunestoneComponents.git@master#runestone
```
[the remark this answers is not in the document]
Comment (sujith-rek) [reply requirements.txt:1]: yes, done

**Review by fzuccolo — APPROVED** [review]

**Pull request merged**